src/vm/customattribute.cpp

```diff
@@ -671,7 +671,6 @@ FCIMPL2(Object*, RuntimeTypeHandle::CreateCaInstance, ReflectClassBaseObject* pC
     CONTRACTL {
         FCALL_CHECK;
         PRECONDITION(CheckPointer(pCaTypeUNSAFE));
-        PRECONDITION(!pCaTypeUNSAFE->GetType().IsGenericVariable()); 
```

Comment:
I believe this precondition is still valid. This is guarding against something like this:

     class G<T>
     {
          [T]   // <-- still disallowed, correct?
          class Inner {}
     }